pkg/plugin/pr-sanitizer/test_fixtures/github_calls/custom_prefix_pr_opened.json

```diff
@@ -0,0 +1,459 @@
+{
+  "action": "edited",
```

Comment:
the file has a name: `custom_prefix_pr_opened.json` but the action is `edited` - it shouldn't influence the logic anyhow, but it's good to keep it consistent